.github/workflows/check-manifest.yml

```diff
@@ -0,0 +1,16 @@
+name: "Check Manifest"
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
We can use setup-python here and use that python:

```yaml
- uses: actions/setup-python@v2
  with:
    python-version: '3.8'
```

Then we can install dependencies:

```yaml
    - name: Install dependencies
      run: |
        python -m pip install --upgrade pip
        pip install check-manifest scipy cython
```

We should also leave a comment about why scipy and cython needs to be installed.